collectors/APICalls/breakpoints.js

```diff
@@ -128,6 +128,7 @@ const breakpoints = [
             {name: 'storage'},
             {name: 'vendor'},
             {name: 'vendorSub'},
+            {name: 'webdriver'},
```

Comment:
We can now check if property was touched by scripts w/o revealing that it exists.

Reply:
So this will be included in the tracker-radar API call data?

Reply:
Yep, everything we add here to the breakpoints ends up, after crawl and processing, in tracker-radar repo.